src/extension.ts

```diff
@@ -76,16 +78,32 @@ export async function activate(context: ExtensionContext) {
     clientOptions,
   );
 
-  client.start();
-  outputChannel.appendLine('..Wasp LSP Server has been started..');
-
-  // Register command to restart waspls
+  outputChannel.appendLine('Starting Wasp LSP Server..');
+  client.start()
+    .then(() => {
```

Comment:
Oh, my bad, I thought this was the last expression in the function and the rest was all happening inside the `then` block.

Reply:
But even then, the difference is if function ends at the moment that client.start() is invoked, or if it waits for it to resolve. So even if this was the last statement/expression, it would still be semantically different to use await here.